build_db.sh: find/use gzipped files, avoid erroneous warnings when shrinking db

jvolkening opened the pull request:
It is great to see that someone is maintaining KrakenUniq again, as it still has features we need not found in Kraken2, etc. I have several patches from the past few years of use which I will submit as separate PRs.

This PR makes two changes to `build_db.sh`:

1. Enable finding and use of gzipped library FASTA files

2. Reduce verbosity of calls to `set_lcas`, which otherwise throw endless warnings to STDERR when using the `--max-db-size` parameter to downsample k-mers (fixes #63)

jvolkening committed:
build_db.sh: find/use gzipped files, avoid erroneous warnings when shrinking db

salzberg commented:
> It is great to see that someone is maintaining KrakenUniq again, as it still has features we need not found in Kraken2, etc. I have several patches from the past few years of use which I will submit as separate PRs.
> 
> This PR makes two changes to `build_db.sh`:
> 
> 1. Enable finding and use of gzipped library FASTA files
> 2. Reduce verbosity of calls to `set_lcas`, which otherwise throw endless warnings to STDERR when using the `--max-db-size` parameter to downsample k-mers (fixes [Large logs #63](https://github.com/fbreitwieser/krakenuniq/issues/63))

Thanks for your comments! Yes, my lab plans to continue maintaining KrakenUniq as long as we can. The postdocs who wrote the code (Florian Breitwieser and Christopher Pockrandt) have moved to industry, but they still help out, and we plan to keep maintaining it in any case - we use it all the time ourselves!

alekseyzimin committed:
Update VERSION

alekseyzimin committed:
Update paper.md

alekseyzimin committed:
Update README.md

Commit:
force preload when building the database for speed

alekseyzimin committed:
Update README.md

boulund committed:
Fix: Add "file" to Dockerfile

read_merger.pl does not work as intended if `file` is not present in the execution environment

alekseyzimin committed:
got rid of file in read_merger.pl

alekseyzimin committed:
Update read_merger.pl

alekseyzimin committed:
Update read_merger.pl

alekseyzimin committed:
Update VERSION

alekseyzimin committed:
Update README.md

alekseyzimin committed:
remove file from Dockerfile

Commit:
small cosmetic change

alekseyzimin committed:
Update krakenuniq

alekseyzimin committed:
Update README.md

alekseyzimin committed:
Update README.md

jvolkening committed:
add parameter to stop build after given stage

jvolkening committed:
Merge branch 'fbreitwieser:master' into build_db_tweaks
